consensus/accept.go

```diff
@@ -108,9 +90,23 @@ func (s *State) validHeader(b Block) (err error) {
 		return
 	}
 
-	// Check that the block is the correct size.
-	if len(encoding.Marshal(b)) > BlockSizeLimit {
-		return LargeBlockErr
+	return
+}
+
+// addBlockToTree takes a block and a parent node, and adds a child node to the
+// parent containing the block. No validation is done.
+func (s *State) addBlockToTree(b Block) (err error) {
```

Comment:
yeah, I think it would make sense in either place, but I moved it here because it was only called by `AcceptBlock`. That way, all the functions called by `AcceptBlock` are defined right next to it.